add-ons/htpasswd-identity-provider/htpasswd-identity-provider.addon.remove

```diff
@@ -0,0 +1,19 @@
+# Name: htpasswd-identity-provider
+# Description: Reverts to default minishift authentication configuration
+# Url: https://docs.openshift.org/3.9/install_config/configuring_authentication.html#AllowAllPasswordIdentityProvider
+# Var-Defaults: MINISHFIT_DATA_HOME=/var/lib/minishift,CONFIG_LOCATION=openshift.local.config/master,MASTER_CONFIG_FILE=master-config.yaml
```

Comment:
@kameshsampath  here you need to put  below because this can be used when removing the addon for 3.9+

```
Required-Vars: CONFIG_LOCATION
```